guides/doc-Content_Management_Guide/topics/Using_ISS.adoc

```diff
@@ -321,6 +336,72 @@ A new Content View called `Import-Library` is created in the target organization
+
+.On the connected {ProjectServer}
+
+* Set the`Default Red Hat Repository download policy` to immediate.
+* Enable the desired content
+* Synchronize the enabled content.
+* For the first export do a `complete library` export so that all the synchronized content are exported (see xref:Using_ISS-Exporting-Library[].)  This will generate content archives that could be imported into one or more disconnected {ProjectServer}s.
+* Export all future updates in the connected {ProjectServer}s incrementally (see xref:Using_ISS-Exporting-Library-Incremental[].). This will generate leaner content archives with only changes from the previous set of updates. So for example if you enabled and synchronized a new repository this exported content archive will only contain the changes.
+
+
+.On the disconnected {ProjectServer}
+
+* Copy the desired exported content archive from the connected {ProjectServer}.
+* Import content to the desired organization (see xref:Using_ISS-Importing-Library[].)
+* You can then manage content via Content Views/Lifecycle Environments appropriately on the Disconnected {ProjectServer}.
+
+==== Disconnected
+
+In this situation you have a {ProjectServer} that is receiving content from an external source like the CDN. The rest of your infrastructure is completely isolated (including the {ProjectServer}.) However the connected {ProjectServer} is not only used as a content store, but also is used manage content. Updates coming from CDN are curated into the desired Content Views/Application Lifecycle Environments. Once the content has been promoted to the desired Application Lifecycle Environment the content can be exported and imported into the disconnected {ProjectServer}.
+
+.On the connected {ProjectServer}
+
+* Set the`Default Red Hat Repository download policy` to immediate.
+* Enable the desired content
+* Synchronize the enabled content.
+* On updates republish the desired Content Views.
+* For the first export do a `complete version` export on the desired Content View version.(see xref:Using_ISS-Exporting-a-Content-View-Version[].)  This will generate content archives that could be imported into one or more disconnected {ProjectServer}s.
+* Export all future updates in the connected {ProjectServer}s incrementally (see xref:Using_ISS-Exporting-a-Content-View-Version-Incremental[].) This will generate leaner content archives with only changes from the previous set of updates. So for example if your Content View had a new repository this exported content archive will only contain the changes.
+
+.On the disconnected {ProjectServer}
+
+* Copy the desired exported content archive from the connected {ProjectServer}.
+* Import content to the desired organization (see xref:Using_ISS-Importing-Content-View-Version[].)
+* This will create a content view version from the exported content archives and them import  content appropriately.
```

Comment:
```suggestion
. Copy the exported content archive from the connected {ProjectServer}.
. Import the content to the organization that you want. 
For more information, see xref:Using_ISS-Importing-Content-View-Version[].
This will create a content view version from the exported content archives and them import  content appropriately.
```